python/src/tests/tasks/test_tasks.py

```diff
@@ -62,11 +48,9 @@ def test_creates_class_on_existent_task(self):
     def test_returns_result_list_with_properly_defined_task(self, valid_task_name):
         results = self.task_factory.submit({"body": {"name": valid_task_name}})
         assert isinstance(results, list)
-        assert self.task_factory.count_matrix.sync.call_count == 2
 
     def test_each_element_in_result_list_is_a_result_object(self):
         results = self.task_factory.submit(
             {"body": {"name": "GetEmbedding", "type": "pca"}}
         )
         assert all(isinstance(result, Result) for result in results)
-        assert self.task_factory.count_matrix.sync.call_count == 2
```

Comment:
I won't worry too much about the tests because I know they passed on the checks, and we are rewriting them either way so this is all Ok as long as it doesn't break.